evm/spec/tables/cpu.tex

```diff
@@ -1,4 +1,55 @@
 \subsection{CPU}
 \label{cpu}
 
-TODO
+The CPU is the central component of the zkEVM. Like any CPU, it reads instructions, executes them and modifies the state (its registers and the memory)
+accordingly. The constraining of some complex instructions (e.g. Keccak hashing) are delegated to the other tables.
+This section will only briefly present the CPU and its columns. Details about the CPU logic will be provided later.
+
+\subsubsection{CPU flow}
+
+An execution run is made of three distinct parts:
+\begin{itemize}
+    \item \textbf{Bootstrapping:} The CPU starts by writing all the kernel code in memory, and hash it. The hash is then compared to a public value shared with
+the verifier to ensure that the kernel code is correct.
+    \item \textbf{CPU cycles:} The bulk of the execution. In each row, the CPU reads the current code at the PC address, and executes it. The current code can be the kernel code,
```

Comment:
If we do not mention the PC before in the specs, maybe say "program counter (PC)"?